libs/elasticsearch/tests/conftest.py

```diff
@@ -18,6 +18,19 @@ def es_client_fx() -> Generator[MagicMock, None, None]:
     yield client_mock()
 
 
+@pytest.fixture
+def es_cache_store_fx(
```

Comment:
maybe `es_embeddings_cache_fx` to make it clear how it's different from `es_cache_fx`?

Reply:
done